py-utils/src/utils/msg_bus/message_bus/message/message.py

```diff
@@ -0,0 +1,29 @@
+from message_bus.constants import MessageConst as msg_c
+from message_bus.message import MessageHeader
+
+class Message(object):

```

Comment:
Python3 does not require you to derive out of object. Please remove derivation.

Reply:
Removed